### ISSUE TITLE
Question: Debug wp-calypso with source-map

### ISSUE THREAD

**Issue by vasileboris** [Automattic/wp-calypso#25719]
#### Steps to reproduce
Good morning. I'm new to wp-calypso codebase and I'm trying to understand how to debug in browser. I noticed that devtool is set to eval in webpack.config.js. If I change to source-map npm build fails with:
1ms after seal
95% emitting CopyPlugin:heavy_multiplication_x: ｢wdm｣: TypeError: Cannot read property 'source' of undefined
   at compiler.hooks.afterEmit.tap.compilation (/media/fastdata/work/wp-calypso/server/bundler/assets-writer.js:54:8)
Is there a reason for not using source-map?

#### What I expected
I expected to see original files when debugging in browser

#### What happened instead
Build failed

### Context / Source

#play

**Comment by bsessions85** [comment]
Hello @vasileboris,  I apologize for the slow response. I asked some of the developers why we aren't using Source Maps and they said that webpack is slow to build a project the size of calypso and enabling source maps is too expensive( see https://webpack.js.org/configuration/devtool/). We used to have a option to enable source maps by a feature flag, but it wasn't used much.

If you're still having trouble debugging, I found some more details in our documentation that may be useful- https://github.com/Automattic/wp-calypso/blob/master/docs/development-workflow.md
